src/Models/Tool/CustomReportResolver.php

```diff
@@ -27,4 +30,14 @@ public function getByName(string $name): ?Config
     {
         return Config::getByName($name);
     }
+
+    public static function getReportsList(User $user = null): array
+    {
+        return Config::getReportsList($user);
+    }
+
+    public static function getAdapter(?stdClass $configuration, Config $fullConfig = null): CustomReportAdapterInterface
```

Comment:
```suggestion
    public static function getAdapter(?stdClass $configuration, ?Config $fullConfig = null): CustomReportAdapterInterface
```

Reply:
Thanks, just realized that we probably wont need `getAdapter` method since it is deprecated and there is another way to get them. 
Same applies to `getReportsList`, it is used in the controller only. In studio we cover it with the hydrator class. I think we can deprecate that method in the core as well.